packages/opentelemetry-core/test/trace/spancontext-utils.test.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+import * as context from '../../src/trace/spancontext-utils';
+
+describe('spancontext-utils', () => {
+  it('should validate invalid values', () => {
```

Comment:
Since this `it` block is just checking constant values, does it make sense to have it?

Reply:
Removed.